src/python_routing_v02/diffusive_utils.py

```diff
@@ -336,21 +394,22 @@ def diffusive_input_data_v02(tw
                             ):
     
     """
-    diffusive_input_data_v02: Builds input data objects for diffusive wave model
-    
-    Args:
-        tw (int): Tailwater segment ID
-        connections (dict): donwstream connections for each segment in the network
-        rconn (dict): upstream connections for each segment in the network
-        reach_list (list of lists): lists of segments comprising different reaches in the network
-        diffusive_parametters (dict): Diffusive wave model parameters
-        ch_geo_dat (pandas DataFrame): geometry parameters for all segments in the network
-        qlats (pandas DatFrame): lateral inflow data for all segments in the network
-        
-    """
+    Build input data objects for diffusive wave model
+    
+    Keyword arguments:
```

Comment:
These are not *technically* python keyword arguments.   They are required *positional* arguments.  I wouldn't confuse the documentation, leave it as Args.  In general, python functions take `*args` as a tuple of positional arguments, and `**kwargs` as a dictionary of key/value arguments.  If these all had some default, i.e `tw=0` in  the function signature, then they would fall under the keyword category.

Reply:
This applies to most if not all of these doc strings

Reply:
Thanks for the clarification - will change!